### ISSUE TITLE
About page going into taskbar

### ISSUE THREAD

**Issue by queze1** [someretical/TheoreticalDiary#3]
![Screenshot 2021-11-18 095131](https://user-images.githubusercontent.com/52340127/142295214-d7954cd8-7557-45af-9bcb-34ee53b93f3c.png)

How to reproduce:
Press about page

**Comment by someretical** [comment]
Please download the latest version and apply the high DPI override as detailed in the readme. Let me know if you are still experiencing this issue.

**Comment by queze1** [comment]
I am still experiencing this issue after downloading the latest version and appling the high DPI override.

**Comment by someretical** [comment]
I have figured out why this issue is occurring. Your screen resolution is 1366 * 768 IIRC. The main window is 800 * 600 and the About window is 600 * 800 in size. It seems your screen is simply too small vertically to fit the entire about window in view. Unfortunately, I am going to have to close this issue since fixing it would mean having to essentially redesign the entire UI (something I already did once and do not want to do again). Plus I am no expert at user interface design. Pull requests are welcome for this issue though.
